pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1277,6 +1277,36 @@ type ObjectStoreSpec struct {
 	// +optional
 	// +nullable
 	Security *SecuritySpec `json:"security,omitempty"`
+
+	// LDAP represents ldap settings
+	// +optional
+	// +nullable
+	LDAP *LDAPSpec `json:"ldap,omitempty"`
```

Comment:
I think this needs to be in a higher level Spec. LDAP and OPA for instance could fall under the same category. Perhaps something `'IdentityManagement`?